apps/core/src/utils/transaction/getObjectChangeSummary.ts

```diff
@@ -37,9 +39,14 @@ export const getObjectChangeSummary = (
         (change) => change.type === 'transferred'
     ) as SuiObjectChangeTransferred[];
 
+    const published = objectChanges.filter(
```

Comment:
@mamos-mysten adding this since we need it on explorer. I believe you will need to add it also.